backend/src/apps/owasp/migrations/0073_activityevent.py

```
@@ -0,0 +1,36 @@
+# Generated by Django 6.0.7 on 2026-07-28 07:54
```

Ruff (0.16.0)

[convention] Missing copyright notice at top of file

(CPY001)